RocketChat/Rocket.Chat.ReactNative
# [FIX] ServerDropdown flashing bigger server icon

## Proposed changes
By changing the current defaultSource image we fix a flashing logo issue that workspace icons have. Both images (remote and local resource) have the same dimensions, but the logo in the remote image is smaller than the local one, meaning that the flashing logo issue solves by simply changing the local resource.

## How to test or reproduce


In order to test this you must be logged in.

1. Open the app
2. Tap at the top where is the name of the workspace
3. Check on the workspaces icons on the left

## Screenshots

### Before
![gif1](https://user-images.githubusercontent.com/36211892/104016317-7da26c80-518c-11eb-9bdc-70062bced11a.gif)

### After
![gif2](https://user-images.githubusercontent.com/36211892/104016335-872bd480-518c-11eb-8725-83f588af3a53.gif)

## Types of changes




## Checklist
